README.md

```diff
@@ -529,50 +529,37 @@ mutation PrepareBid($input: prepareBidInput!) {
     authorizations {
       fingerprint
       request {
-        ... on StarkexLimitOrderAuthorizationRequest {
-          vaultIdSell
-          vaultIdBuy
-          amountSell
-          amountBuy
-          tokenSell
-          tokenBuy
-          nonce
-          expirationTimestamp
-        }
+        ...AuthorizationRequestFragment
       }
     }
   }
 }
+${authorizationRequestFragment}
 ```
 
-1. Sign all `AuthorizationRequest` objects and build the `bidInput` argument.
+`AuthorizationRequestFragment` is defined in [authorizations.js](authorizations.js).
```

Comment:
```suggestion
`AuthorizationRequestFragment` is defined in [authorizations.js](./authorizations.js).
```

Reply:
Any reason to have it root level and not into examples?

Reply:
It was in examples first, but I felt like it's more of a a helper code than an example. Can move it there if you prefer. Let me know.

Reply:
I see, I'd be inclined to move it into examples as the js project is setup there with the other files. We can reference `./examples/authorizations.js` from the README

Reply:
ok